operator/src/main/java/oracle/kubernetes/operator/EventConstants.java

```diff
@@ -54,4 +53,22 @@
   String ROLL_REASON_DOMAIN_RESOURCE_CHANGED = "domain resource changed";
   String ROLL_REASON_WEBLOGIC_CONFIGURATION_CHANGED
       = "WebLogic domain configuration changed due to a Model in Image model update";
+  String DOMAIN_INVALID_ERROR = "Domain validation error";
+  String TOPOLOGY_MISMATCH_ERROR = "Domain resource and WebLogic domain configuration mismatch error";
+  String INTROSPECTION_ERROR = "Introspection error";
+  String KUBERNETES_ERROR = "Kubernetes Api call error";
+  String SERVER_POD_ERROR = "Server pod error";
+  String REPLICAS_TOO_HIGH_ERROR = "Replicas too high";
+  String INTERNAL_ERROR = "Internal error";
+  String ABORTED_ERROR = "Domain processing is aborted";
+  String ABORTED_ERROR_SUGGESTION = "The domain will not be retried unless the problem is corrected.";
```

Comment:
How about something like `The reported problem should be corrected, and the domain will not be retried until the domain resource is updated.`

Reply:
done. thanks